src/env-inl.h

```diff
@@ -146,7 +146,7 @@ inline bool Environment::AsyncHooks::pop_ids(double async_id) {
   if (uid_fields_[kCurrentAsyncId] != async_id) {
     fprintf(stderr,
             "Error: async hook stack has become corrupted ("
-            "actual: %'.f, expected: %'.f)\n",
+            "actual: %f, expected: %f)\n",
```

Comment:
> If you prefer, casting to uint64_t should be safe here.

I'll read a bit about `fprintf` on windows a think about it.
IMHO for this PR `%.f` works